package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/fastify/fastify-sensible#readme",
   "devDependencies": {
-    "fastify": "^1.13.2",
+    "fastify": "next",
```

Comment:
Use the tag is ok, but we must remember to update this with the version number once we release the final version of v2, because the tag will become `latest`.

Reply:
You are right, using `^2.0.0-rc.2` is safer